vortex-array/src/array/byte_bool/stats.rs

```diff
@@ -0,0 +1,41 @@
+use std::collections::HashMap;
+
+use vortex_error::VortexResult;
+
+use super::ByteBoolArray;
+use crate::{
+    stats::{ArrayStatisticsCompute, Stat, StatsSet},
+    validity::{ArrayValidity, LogicalValidity},
+    ArrayDType, ArrayTrait, AsArray,
+};
+
+impl ArrayStatisticsCompute for ByteBoolArray {
+    fn compute_statistics(&self, stat: Stat) -> VortexResult<StatsSet> {
+        if self.is_empty() {
+            return Ok(StatsSet::new());
+        }
+
+        match self.logical_validity() {
+            LogicalValidity::AllValid(len) => Ok(all_true_bool_stats(len)),
```

Comment:
I don't think this works? Just because all values are non-null, doesn't mean all values are true.

Reply:
100%